invopop/invopop.go

```diff
@@ -198,3 +198,16 @@ func (c *Client) patch(ctx context.Context, path string, in, out any) error {
 	}
 	return re.handle(res)
 }
+
+func (c *Client) delete(ctx context.Context, path string, result interface{}) error {
```

Comment:
Using `any` is the convention now.
```suggestion
func (c *Client) delete(ctx context.Context, path string, result any) error {
```